teachertool/src/components/styling/Toasts.module.scss

```diff
@@ -0,0 +1,133 @@
+.toast-container {
+    display: flex;
+    gap: 0.5rem;
+    flex-direction: column-reverse;
+    align-items: flex-end;
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    margin-bottom: 2rem;
+    margin-right: 1rem;
+    z-index: 50;
+    pointer-events: none;
+}
+
+.toast {
+    display: flex;
+    flex-direction: column;
+    margin-right: 0;
+    border: none;
+    border-radius: 0.25rem;
+    box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06);
+    overflow: hidden;
+    pointer-events: none;
+
+    &.success {
+        background-color: #81E6D9;
```

Comment:
These colors should be declared as vars in `theme/themepacks.less`, and overridden in targets.

Reply:
Eventually we will want to define color vars for toast border/background/text so that we can implement high-contrast toasts. But background color only is fine for this iteration.

Reply:
Also fine to push this to a future PR, track it as a todo.